'use strict';

/*
 * try/catch
 * Помилки фази парсинга опрацювати неможна. Тільки помилки фази виконання.
 * Об'єкт помилки
 * try/catch працює тільки з синхронним кодом
 * Опрацювання помилок в асинхронному коді
 */

// console.log('First');

// setTimeout(() => {
//   try {
//     console.log(userName);
//   } catch (error) {
//     console.log(error);
//   }
// }, 0);

// console.log('Second');
